packages/frontend/src/components/tvl-breakdown/table/TokenAmountCell.tsx

```diff
@@ -23,28 +24,30 @@ export function TokenAmountCell(props: TokenAmountCellProps) {
       : ''
 
   return props.forCanonical && props.escrows ? (
-    <div
-      className="Tooltip flex flex-col items-end gap-2 text-xs font-medium"
-      title="Circulating supply"
-    >
-      {formatLargeNumberWithCommas(Number(props.amount))}
-      {props.escrows.length > 1 &&
-        props.escrows.map((escrow) => (
-          <div
-            key={escrow.escrowAddress.toString()}
-            className="MultipleEscrowsHidden hidden font-normal text-black/80 dark:text-white/80"
-            data-token={props.assetId}
-          >
-            {formatLargeNumberWithCommas(Number(escrow.amount))}
-          </div>
-        ))}
-    </div>
+    <Tooltip className="flex flex-col items-end gap-2 text-xs font-medium">
+      <TooltipTrigger>
+        {formatLargeNumberWithCommas(Number(props.amount))}
+        {props.escrows.length > 1 &&
+          props.escrows.map((escrow) => (
+            <div
+              key={escrow.escrowAddress.toString()}
+              className="MultipleEscrowsHidden hidden font-normal text-black/80 dark:text-white/80"
+              data-token={props.assetId}
+            >
+              {formatLargeNumberWithCommas(Number(escrow.amount))}
+            </div>
+          ))}
+      </TooltipTrigger>
+      <TooltipContent>Circulating supply</TooltipContent>
+    </Tooltip>
```

Comment:
Something is broken here:
![image](https://github.com/l2beat/l2beat/assets/77489537/a052c3f1-206c-4d84-9f0e-e9c3883fe879)